src/bit_vector.jl

```diff
@@ -30,29 +30,29 @@ end
 
 # blob interface
 
-function Base.size(blob::Blob{BlobBitVector})
+@inline function Base.size(blob::Blob{BlobBitVector})
```

Comment:
How come such aggressive use of `@inline` hints is required?  Surely the compiler would inline a function like this one without the hint?  How have you confirmed that the changes here represent an improvement?

Reply:
Well, you have a point and the same question occurred to me and tried to confirm it. The benchmarks in a user project do not show any runtime performance improvement and increase compilation time. Then, we can drop this PR for now.